lib/support/har/index.js

```diff
@@ -70,7 +70,8 @@ function addExtrasToHAR(
       'ContentfulSpeedIndex',
       'VisualProgress',
       'ContentfulSpeedIndexProgress',
-      'PerceptualSpeedIndexProgress'
+      'PerceptualSpeedIndexProgress',
+      'KeyColorFrames'
```

Comment:
This is metrics added to the HAR file so HAR viewers can draw the metrics in the HAR waterfall, would the keycolorframes and any value here?

Reply:
It could be. I'm not familiar with the tool and how I would need to massage the data to be compatible with the viewer, so I just excluded it.

Reply:
Sorry what I meant was I think you can safely remove it from there, thanks :)